code/modules/paperwork/photocopier.dm

```diff
@@ -239,6 +243,71 @@
 	return p
 
 
+
+
+//////Premium(upgraded) photocopier/////
+
+
+
+
+/obj/structure/machinery/photocopier/wyphotocopier
+	name = "photocopier"
+	icon = 'icons/obj/structures/machinery/library.dmi'
+	icon_state = "bigscannerpro"
+	desc = "A photocopier used for copying...you know, photos! Also useful for copying documents in paper. This specific model have been manufactured by Weyland-Yutani in a more modern and robust frame than the average photocopiers you see being sold by smaller companies, while also carrying some of the most advanced technologies on the area of paper-printing, such as bigger toner economy and much higher printing capabilities. All that makes it the favorite among consumers that need to print high amounts of paperwork on the performance of their daily duties."
+	anchored = TRUE
+	density = TRUE
+	use_power = USE_POWER_IDLE
```

Comment:
You don't need to redefine values from parent type if they're going to be the same ones